website/docs/r/group.markdown

```diff
@@ -15,17 +15,35 @@ Manages a Group within Azure Active Directory.
 
 ## Example Usage
 
+*Basic example*
+
 ```hcl
 resource "azuread_group" "my_group" {
   name = "MyGroup"
 }
 ```
 
+*A group with members*
+
+```hcl
+resource "azuread_user" "my_user" {
+  display_name          = "John Doe"
+  password              = "notSecure123"
+  user_principal_name   = "john.doe@terraform.onmicrosoft.com"
+}
+
+resource "azuread_group" "my_group" {
+  name = "MyGroup"
+  members = [ azuread_user.my_user.id /*, more users */ ]
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
-* `name` - (Required) The display name for the Group.
+* `name` - (Required) The display name for the Group. Changing this forces a new resource to be created.
+* `members` (Optional) A set of users who should be members of this Group.
```

Comment:
This should support both groups and SPs as well?

Reply:
AS well a warning to not use the `azuread_group_member` at the same time might be handy here